assets/model_monitoring/components/src/feature_importance_metrics/compute_feature_importance.py

```diff
@@ -92,7 +97,12 @@ def create_lightgbm_model(X, y, task_type):
         lgbm = LGBMRegressor(boosting_type='gbdt', learning_rate=0.1,
                              max_depth=5, n_estimators=200, n_jobs=1, random_state=777,)
 
-    model = lgbm.fit(X, y)
+    try:
+        model = lgbm.fit(X, y)
+    except ValueError as e:
+        if "Unknown label type: 'continuous'" in str(e):
```

Comment:
Is it also possible to have scenario where target value is numerical but the tasktype is classification? If yes, can we add the validation as well?

Reply:
sorry not sure I understand - this PR is indeed for this bugfix, numerical (continuous) column for classification task type is raising the wrong error, I made it a user error as suggested